src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -344,7 +213,7 @@ public void should_fail_when_there_is_no_case_with_given_legacy_id() throws Exce
     }
 
     @Test
-    public void should_fail_when_there_are_multiple_cases_with_given_legacy_id() throws Exception {
+    void should_fail_when_there_are_multiple_cases_with_given_legacy_id() throws Exception {
```

Comment:
it would be so much easier to find relavant changes if those were not on this PR ;)